solr/config/schema.xml

```diff
@@ -248,6 +248,9 @@
 
     <!-- exhibits fields -->
     <field name="full_text_search" type="text" indexed="true" stored="true" termVectors="true" termPositions="true" termOffsets="true" multiValued="true" />
+    <field name="full_text_search_en" type="text_en" indexed="true" stored="true" termVectors="true" termPositions="true" termOffsets="true" multiValued="true" />
```

Comment:
Not totally sure about the naming of these fields.  It seems like the right approach to make the various `full_text_search_*` fields for each language and then copy from full_text_search into them.  Totally willing to entertain other options.